### DISCUSSION TITLE
is there a way to publish datacook as web assets?

### DISCUSSION THREAD

**Discussion by yorkie** [imgcook/datacook#22] [Ideas]
Hey @imgcook/data-scientist 

Thanks to @steveoni's work to achieve Word2Vector in pure JavaScript, then I wanna write a tutorial about it at deno/browser environment, so do you guys have any idea to publish our asset? Alternatives, as I know, are bower, cdnjs, or github?

**Comment by risenW** [comment]
@yorkie I know of http://jsdelivr.com/ offers free host once a package is published on npm.
